addons/udes_stock/models/stock_location.py

```diff
@@ -198,6 +198,11 @@ def _validate_inventory_adjustment_request(self, request):
 
         product = Product.get_product(int(request['product_id']))
 
+        # Ignore conflicts of tracked vs untracked if we are removing existing
+        # stock
+        if request['quantity'] == 0:
```

Comment:
Is this to cover the case when setting existing stock to zero for the old tracked/ untracked quant? Does this not give us a problem when doing a normal stock check (regardless of tracked) and setting quantity to zero for a location?

Reply:
No, because quantity to zero is only set when something is not accounted during stock check.